docs/index.rst

```diff
@@ -97,6 +97,20 @@ like buffering:
     >>> secondpart = b.read()
 
 
+The response can be treated as a file-like object.
+A file can be downloaded directory to a local file in a context without
+being saved in memory.
+
+.. doctest ::
+
+    >>> import urllib3
```

Comment:
For consistency with other examples, let's skip the urllib3 import? (I think the narrative assumed the examples continue from previous contexts.)